test/specs/api/attachments.spec.js

```diff
@@ -37,4 +41,17 @@ describe('Attachments API', () => {
             });
         });
     });
+
+    describe('#delete', () => {
+        it('should call http', () => {
+            const fullUrl = `${serviceUrl}/v1/attachments/remove`;
+            const body = {
+                mediaUrl: 'http://some-url.com/document.pdf'
+            };
+
+            return api.delete(body).then(() => {
```

Comment:
How are you going to document this method call? 

Like this
```
api.delete('http://some-url.com/document.pdf')
```

or this?
```
api.delete({mediaUrl: 'http://some-url.com/document.pdf'})
```

Both will work, but your test should match what we document.

Reply:
In that case ✅